packages/ui/src/components/va-file-upload/VaFileUpload.vue

```diff
@@ -9,7 +9,7 @@
         class="va-file-upload__field__text"
         v-if="dropzone"
       >
-        Drag’n’drop files or
+        <slot name="dropZoneLabel">Drag’n’drop files or</slot>
```

Comment:
Having prop for label is more appropriate. We can use [Components Config]( https://vuestic.dev/en/services/components-config) for i18n for all components :) 

When I spoke about slot, I meant a slot that can be used to change overall file-upload visual (to replace default button and label text). Sorry for that. Let's better have props that can be changed in global config.